hyrisecockpit/api/app/metric/service.py

```diff
@@ -0,0 +1,85 @@
+"""Services for metrics."""
+from time import time_ns
+from typing import Dict, List, Union
+
+from hyrisecockpit.api.app.connection_manager import StorageConnection
+from hyrisecockpit.api.app.historical_data_handling import (
+    get_historical_metric,
+    get_interval_limits,
+)
+from hyrisecockpit.api.app.shared import _get_active_databases
+
+from .interface import TimeIntervalInterface
+from .model import DetailedQueryEntry, DetailedQueryInformation
+
+
+class MetricService:
+    """Services of the Control Controller."""
+
+    @staticmethod
+    def get_data(
+        time_interval: TimeIntervalInterface, table_name: str, column_names: List[str]
```

Comment:
I believe we should be consistent with our refactoring and use models within the service.

Schema: used with `@accepts` and `@responds` for documenting and validating the API,
Interface: Type safety at the controller input level (`input: InputInterface = ...`)
Model: Should be the default

```suggestion
        time_interval: TimeInterval, table_name: str, column_names: List[str]
```